diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/IndentationRule.kt

```diff
@@ -225,9 +227,22 @@ class IndentationRule(configRules: List<RulesConfig>) : DiktatRule(
     private fun fixStringLiteral(whiteSpace: PsiWhiteSpace, expectedIndent: Int) {
         val textIndent = " ".repeat(expectedIndent + INDENT_SIZE)
         val templateEntries = whiteSpace.node.treeNext.firstChildNode.getAllChildrenWithType(LITERAL_STRING_TEMPLATE_ENTRY)
-        templateEntries.forEach {
-            if (!it.text.contains("\n") && it.text.isNotBlank()) {
-                (it.firstChildNode as LeafPsiElement).rawReplaceWithText(textIndent + it.firstChildNode.text.trim())
+        templateEntries.forEach { node ->
+            if (!node.text.contains("\n") && node.text.isNotBlank()) {
+                val correctedText = StringBuilder()
+                when {
```

Comment:
very overloaded logic!
need to split it somehow (methods?)

Reply:
Moved it to method